es/authentication_authorization/README.md

```diff
@@ -34,18 +34,18 @@ No te olvides del decorador encima de `post_edit`, `post_remove`, `post_draft_li
 
 Ahora podemos intentar hacer muchas cosas mágicas para implementar usuarios y contraseñas y autenticación, pero hacer esto correctamente es complicado. Como django viene con "baterías incluidas", alguien ya ha hecho el trabajo duro por nosotros, así que vamos a utilizarlas.
 
-En `mysite/urls.py` agrega una url `url(r'^accounts/login/$', views.login, name='login')`. Así el archivo debería verse similar a este:
+En `mysite/urls.py` agrega una url `path('accounts/login/', views.login, name='login')`. Así el archivo debería verse similar a este:
 
 ```python
-from django.conf.urls import include, url
+from django.urls import path, include
```

Comment:
For the sake of keeping things similar to the original content, let's switch this back to:

```
from django.urls import include, path
```